scripts/prepare_sector_network.py

```diff
@@ -2007,10 +2010,11 @@ def limit_individual_line_extension(n, maxext):
         snakemake = mock_snakemake(
             'prepare_sector_network',
             simpl='',
-            clusters=48,
+            clusters='45,
```

Comment:
```suggestion
            clusters=45,
```

sorry, that commit was a bit odd.